### PR TITLE
use roxygen2 for imports, fix jaspBase imports, and cleanup DESCRIPTION

### PR DESCRIPTION
**Pull request by vandenman**
In addition, there is now this error:

image

which appears when refreshing the ASRM Jasp file in the data library.

Before there was this:

image

which is worse.

The error originates from `Bayesrel:::covSamp`, @juliuspf can you fix that in a different PR?

### CODE REVIEW

**Comment by vandenman** [comment]
I removed `lme4` because of this note in the R CMD check:
```r
N  checking dependencies in R code (597ms)
   Namespace in Imports field not imported from: ‘lme4’
     All declared Imports should be used.
```
but `psych` uses it and has it under `Suggests` so it's not installed by default.. sigh..

File: R/uniDimBayesianCalcCoeff.R [hunk 1]
```diff
@@ -705,7 +705,7 @@
                                                 a0 = options[["igShape"]], b0 = options[["igScale"]],
                                                 m0 = options[["loadMean"]]), silent = TRUE)
 
-    if (model[["pairwise"]] && inherits(tmp_out, "try-error")) {
+    if (model[["pairwise"]] && inherits(res_null, "try-error")) {
```
Comment (juliuspf) [inline R/uniDimBayesianCalcCoeff.R:708]: why this change?

**Review by juliuspf — COMMENTED** [review]
Looks all standard to me. As discussed, I cannot check the changes with JASP... Lets wait and see

**Comment by juliuspf** [comment]
I think the error you mention originates here: https://github.com/vandenman/jaspReliability/blob/fixup_imports/inst/qml/uniDimBayesian.qml#L389-L397 
The formula 1e-10 does not get parsed properly and the actual value for the inverse Wishart scale value returns NULL. Then the computation fails. Would you know how to prevent this?

**Comment by vandenman** [comment]
I can't replicate the error with jaspTools though, what makes you think this?

**Comment by juliuspf** [comment]
the test dont catch this error. the value of `iwScale` is set for all the other tests, but the last one that uses the asrm data. However, the last test only estimates omega and none of the coefficients that use `iwScale`, if you change the test to compute alpha, you will get the error.

**Comment by vandenman** [comment]
Actually this is related to progress bars. It should be fixed by https://github.com/jasp-stats/jaspBase/pull/70

**Comment by vandenman** [comment]
So I think the error is no longer relevant, and this can be merged (unless you have any comments).

**Comment by juliuspf** [comment]
If you are 100% certain about this.... I still would like your answer to the question about the code above https://github.com/jasp-stats/jaspReliability/pull/116#discussion_r935683154

**Comment by juliuspf** [comment]
although, on second thought, the PR is good, the error would then be unrelated to it, so I guess we could always fix it later if it persists.

**Review by juliuspf — APPROVED** [review]
